ground/gcs/src/plugins/config/modules.ui

```diff
@@ -1083,7 +1170,7 @@
            <rect>
             <x>0</x>
             <y>0</y>
-            <width>360</width>
+            <width>392</width>
```

Comment:
@tracernz This created the merge conflict - not sure if this is still the correct value.